pom.xml

```diff
@@ -292,6 +292,13 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>junit</artifactId>
             <version>1.24</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>scm-api</artifactId>
+            <version>2.0.8</version>
+            <type>jar</type>
```

Comment:
Hi, why did you have to include the `scm-api`? Was it to fix an issue running tests?

Reply:
Hi @kinow, yes I assumed this from one of the error messages while getting the build to run. Also forgot to mention, that I built the project with `mvn package -Dmaven.test.skip=true` for the same reason. Thus all warnings and errors seem to relate to testing/doc, where I have no clue how to fix.

Build environment is:
* CentOS 7
* OpenJDK 1.8
* Maven 3.9.6 (manually installed)

Thanks for your valuable work and superfast response!